client/index.html

```diff
@@ -205,6 +205,13 @@ <h1 class="title">
 							<div class="col-sm-12">
 								<h1 class="title">Settings</h1>
 							</div>
+							<% if (displayConnection === true) { %>
```

Comment:
When the option is disabled, all you do is hide this in HTML, but all the socket/dns lookups are still performed.

Also, I'm not entirely sure the option is even needed, perhaps just always have it work in private mode.